### PR TITLE
creation of thucyd-0.1.2 conda-forge recipe

### PR DESCRIPTION
**Pull request by jaydamask**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

I confirm that I am willing to be listed as a maintainer of the conda-forge/thucyd package. 

Since this is a new package, the src code and builds are available here: https://gitlab.com/thucyd-dev/thucyd. 

I look forward to having this package hosted on conda-forge!

-Jay

### CODE REVIEW

File: recipes/thucyd/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  sha256: "e3c0c736746b0ac26b93fa8d2fcd2d90394adb080d60a84144aabf9f83325381"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - numpy >=1.14
```
Comment (chrisburr) [inline recipes/thucyd/meta.yaml]: I don't think you need `nump`y here:
```suggestion
```

File: recipes/thucyd/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    - python >=3.5
+
+test:
+  imports:
+    - thucyd
+
+about:
+  home: "https://gitlab.com/thucyd-dev/thucyd"
+  license: "Apache 2.0"
```
Comment (chrisburr) [inline recipes/thucyd/meta.yaml]: Using the SPDX names of the licences is useful for consistency:
```suggestion
  license: Apache-2.0
```

File: recipes/thucyd/meta.yaml (hunk 1 above)
Comment (jaydamask) [reply recipes/thucyd/meta.yaml]: Hi Chris, why do you say this? In order to run: numpy > 1.14 I'd need that on the host. On line 13 of [my source](https://gitlab.com/thucyd-dev/thucyd/blob/master/src/thucyd/eigen/impl.py) I import numpy. 

I suppose I'm missing something about the conda-forge build. Please let me know.

File: recipes/thucyd/meta.yaml (hunk 2 above)
Comment (jaydamask) [reply recipes/thucyd/meta.yaml]: Sure, I'll change this.

Comment (jaydamask) [reply recipes/thucyd/meta.yaml]: Let me know about the comment above and then I'll push the resolution changes.

File: recipes/thucyd/meta.yaml (hunk 1 above)
Comment (chrisburr) [reply recipes/thucyd/meta.yaml]: The `host` dependencies are those required to install the package (i.e. run your `setup.py` script). If you don't import numpy there (or use it's headers for compiling an extension) then you don't need it in `host`.

Comment (jaydamask) [reply recipes/thucyd/meta.yaml]: Oh, I see. That's clear. I'll remove -- thanks.

**Comment by jaydamask** [comment]
@chrisburr Chris, I see from other projects that I can commit my own README.md page, although I didn't before b/c I didn't know. What's best practice? Shall I add my own README.md to my next push? Just let me know. Thx -Jay

File: recipes/thucyd/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+test:
+  imports:
+    - thucyd
+
+about:
+  home: "https://gitlab.com/thucyd-dev/thucyd"
+  license: Apache-2.0
+  license_family: "APACHE"
+  license_file: LICENSE
+  summary: "Scientific library for realtime signal processing and eigenanalysis of evolving systems."
```
Comment (chrisburr) [inline recipes/thucyd/meta.yaml]: The README is automatically generated by conda-smithy each time the package is re-rendered so you should never modify it directly. If you want to customise it you should modify the `about` second of the metadata, potentially including a `description`:

```suggestion
  summary: "Scientific library for realtime signal processing and eigenanalysis of evolving systems."
  description : |
     This awesome package does:
         * This
         * and this
         * and this

     If you use this for a publication please cite xxx.
```

Comment (jaydamask) [reply recipes/thucyd/meta.yaml]: Ok, got it. Thanks.

File: recipes/thucyd/meta.yaml (hunk 1 above)
Comment (jaydamask) [reply recipes/thucyd/meta.yaml]: committed and built.

File: recipes/thucyd/meta.yaml (hunk 2 above)
Comment (jaydamask) [reply recipes/thucyd/meta.yaml]: committed and built.

**Pull request merged**